package.json

```diff
@@ -30,7 +30,9 @@
     "type:check:tests:0.72+": "./scripts/test-ts.sh __typetests__/0.72+",
     "type:check:tests:legacy": "./scripts/test-ts.sh __typetests__/legacy",
     "type:check:all": "yarn type:check:src && yarn type:check:plugin && ./scripts/test-ts.sh app/src/App.tsx __typetests__/common __typetests__/0.72+ __typetests__/legacy",
-    "prepare": "yarn plugin && bob build && husky install && yarn app",
+    "build": "yarn build:plugin && bob build && husky install && yarn build:app ",
+    "build:app": "cd app && yarn install && yarn build && cd ..",
+    "build:plugin": "cd plugin && yarn install && yarn build && cd ..",
```

Comment:
Not sure if `&& cd ..` at the end are required; as far as I remember yarn restores the original current working directory.

Reply:
You are right, it's not necessary.